packages/flame_console/lib/src/repository/debug_command_test.dart

```diff
@@ -0,0 +1 @@
+
```

Comment:
Remove?

Reply:
Or implement it hahha? I probably forgot to finish the test 😅